### PR TITLE
Add service to get document in a specific version

### PR DESCRIPTION
**Pull request by tsauerwein**
Request: "/waypoints/{id}/{lang}/{version_id}"
for example: http://localhost:6544/waypoints/642179/fr/1020298

Example response:

    {
        "document": {...},
        "version": {
            "username": "lazare",
            "comment": "Edit in fr",
            "user_id": 6206,
            "written_at": 1434325861,
            "version_id": 1020298
        }
    }

### CODE REVIEW

**Comment by asaunier** [comment]
Thanks! Looks good to me!

**Comment by asaunier** [comment]
@gberaudo any objection?
If not, feel free to merge soon @tsauerwein this PR is needed for viewing an archived doc version as well.

**Comment by asaunier** [comment]
> Request: "/waypoints/{id}/{lang}/{version_id}"

By the way I notice a little inconsistency: for getting an archived version we have a dedicated route with ``{lang}`` in matchdict whereas in the standard view, "lang" should be passed as a GET ``l`` param.

**Comment by gberaudo** [comment]
postmerge +1

**Comment by tsauerwein** [comment]
> By the way I notice a little inconsistency: for getting an archived version we have a dedicated route with {lang} in matchdict whereas in the standard view, "lang" should be passed as a GET l param.

Yes, good point.

File: c2corg_api/views/document.py [hunk 1]
```diff
@@ -336,6 +338,40 @@ def _check_update_type(self, document, old_versions):
                 'trying do update the document with the same content')
         return (update_types, changed_langs)
 
+    def _get_version(self, clazz, schema, adapt_schema=None):
+        id = self.request.validated['id']
+        lang = self.request.validated['lang']
+        version_id = self.request.validated['version_id']
+        document = self._get_in_lang(
```
Comment (gberaudo) [inline c2corg_api/views/document.py]: @tsauerwein are you sure the return document is the one form the requested version?
Shouldn't it be looked in the archives?

Comment (tsauerwein) [inline c2corg_api/views/document.py]: You are right!

**Pull request merged**